lib/active_shipping/carriers/ups.rb

```diff
@@ -424,18 +409,14 @@ def build_shipment_request(origin, destination, packages, options={})
                   # Type '01' means 'Transportation'
                   # This node specifies who will be billed for transportation.
                   xml.Type('01')
-                  xml.BillShipper do
-                    xml.AccountNumber(options[:origin_account])
-                  end
+                  build_billing_info_node(xml, options)
```

Comment:
Yes, putting BillThirdParty here seems to do the same thing as having it outside of the ItemizedPaymentInformation except if you put it here you can also add a node to bill the same account for the Duties and Taxes via the BillThirdPartyConsignee node.